source/documentation/standards/how-to-get-a-domain-name.html.md.erb

```diff
@@ -0,0 +1,52 @@
+---
+title: How to get, register or manage a domain name
+last_reviewed_on: 2021-09-29
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+The MOJ centrally registers and maintains domains for use across the department.
+Domains used within the MOJ must follow the [naming domains]
+(naming-domains.html) standards and must be registered by the [MOJ's Domains
```

Comment:
I'd prefer to refer to the Team as Operations Engineering

Reply:
That applies throughout the document.

Reply:
@AntonyBishop Have updated that now.